### DISCUSSION TITLE
Exit node support?

### DISCUSSION THREAD

**Discussion by pcmike** [almeidapaulopt/tsdproxy#112] [Q&A]
Is there a way to specify an exit node a certain service should use via labels? Further more, is there a way to ensure that said exit node is actually available and if it’s not, then the service can’t start and/or route to the internet?

**Comment by NotSwedishBacon** [comment]
I can see this being particularly useful to exit via mullvad. has my vote

**Comment by mayorgergich** [comment]
Would love for this to be added, especially being able to send containers to specific mullvad exit nodes.

**Comment by sashavrg** [comment]
yes please, would love to route select containers through mulvad exit nodes! seconded

**Comment by Drallas** [comment]
As a Proton Duo user, I prefer not to pay extra for VPN services like Mullvad. Since I already have multiple Tailscale exit nodes, it would be ideal if the TSDSProxy container could support (a flag exit-node: name) routing traffic via a selected exit node.

This is especially useful for bypassing IP geo-fences: running tsdproxy inside a container with exit-node support would let me route traffic through a clean node...

**Comment by patrick-theprogrammer** [comment]
I also think this would be a cool feature, but I’m not sure that tsnet supports it 😕 https://github.com/tailscale/tailscale/issues/8553
